jaxrs/src/test/java/io/airlift/jaxrs/TestJacksonJsonMapper.java

```diff
@@ -67,13 +69,12 @@ private static void assertRoundTrip(String value)
 
     @Test
     public void testJsonEofExceptionMapping()
-            throws IOException
     {
         try {
             jsonMapper.readFrom(Object.class, Object.class, null, null, null, new ByteArrayInputStream("{".getBytes(UTF_8)));
             fail("Should have thrown an Exception");
         }
-        catch (JsonParsingException e) {
+        catch (UnexpectedEndOfInputException e) {
```

Comment:
This seems wrong, as it's malformed JSON, not an IO error.

Reply:
Fixed